tests/wpt/css-tests/cssom-1_dev/html/index-002.htm

```diff
@@ -30,7 +30,7 @@
       ],
 
       'overflow': [
-        ['overflow-x: scroll; overflow-y: hidden;', 'overflow: scroll hidden;'],
+        ['overflow-x: scroll; overflow-y: hidden;', 'overflow-x: scroll; overflow-y: hidden;'],
```

Comment:
Just checked this, and this change seems to match what both firefox and chromium are doing, so it seems to me that we should try to upstream this or file an issue about this.

Can you remove your changes from here and open an issue or a PR against the upstream repo?